server/migrations/20200410010141-add-flagging-fields-store-updates.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return Promise.all([
+        queryInterface.addColumn("StoreUpdates", "flag", {
```

Comment:
these fields will also need to be added to the model definition inside `models`. also can you verify that the insert and query are working?

Reply:
Does it? I built the DB from scratch and it worked. These are incremental additions on the base model, you shouldn't need to add this to the base model.

Reply:
I'll check the query, I didn't verify that. The insert should work, since these have a default value.

Reply:
the models are used by the JS code and need to be updated.

Reply:
If not added to the model, it won't be available for cascading updates that we are doing.

Reply:
My bad, I was confusing it with the migrations folder. Added now!

Reply:
How do I go about testing the query?

Reply:
you could try adding in some data by running the frontend pointing to the server (change this inside `api.js`), and then see if the frontend displays the results on the homepage. thinking if we can have an easier testing story as well - feedback welcome!